src/admin/index.js

```diff
@@ -173,6 +173,25 @@ module.exports = ({
     })
   }
 
+  /**
+   * @param {string} topic
+   */
+
+  const fetchTopicOffsets = async topic => {
```

Comment:
If we had a method that accepted a list of topics and returned a list of partition-offsets, what would we call that one?

I'm thinking about what the use cases for this would be, and I'm guessing a common usage would be to calculate lag. In that case, I would assume that it's not uncommon to be consuming more than one topic, in which case it would be nice to avoid people doing the following, since we could get information for all topics with a single call to the broker:

```javascript
const offsets = await Promise.all(async topics => admin.fetchTopicOffsets(topic))
```

Maybe we should have this method receive a list of topics instead? It feels a bit redundant to have something like:

```javascript
const fetchMultipleTopicOffsets = async (topics = []) => { // do stuff }
const fetchTopicOffsets = async topic => fetchMultipleTopicOffsets([topic])
```

Reply:
@Nevon personally i am fan of atomic operations , ` Promise.all` and `fetchMultipleTopicOffsets` will do exactly the same background works to get the data.
its all depends on situation and what user needs.

Reply:
You're right. I didn't look very closely at what calls we would actually need to make, but they are indeed all scoped by a single topic, so in the end we would be doing the exact same work. If so, I think your API feels much nicer.